test/generators/suspenders/install/web_generator_test.rb

```diff
@@ -39,7 +39,7 @@ class WebGeneratorTest < Rails::Generators::TestCase
         end
 
         test "raises if Node is unsupported" do
-          Object.any_instance.stubs(:`).returns("v19.9.9\n")
+          Generators::Install::WebGenerator.any_instance.stubs(:node_version).returns("19.9.9")
```

Comment:
The only downside to this is that it no longer checks that the regex working to strip the `v` and `\n`.  We could always make a helper to wrap `Object.any_instance.stubs(:`).returns` so that it's more clear what its intent is.

Reply:
@stevepolitodesign I've pushed up two changes:

- our work to match on the entire string in dffb9033d317ed4c52377a42b78c66d764cb7797
- our work to unit test the `node_version` method, especially with regard to the alpha-stripping regex in 83097d81ba1ca22a79946bc8068749c8c2b5c46b